Validation/HcalHits/src/SimG4HcalValidation.cc

```diff
@@ -223,13 +229,10 @@ void SimG4HcalValidation::init() {
   count = 0;
 }
 
-void SimG4HcalValidation::update(const BeginOfJob *job) {
+void SimG4HcalValidation::beginRun(edm::EventSetup const &es) {
   // Numbering From DDD
-  edm::ESHandle<HcalDDDSimConstants> hdc;
-  (*job)()->get<HcalSimNumberingRecord>().get(hdc);
-  const HcalDDDSimConstants *hcons = hdc.product();
-  edm::LogVerbatim("ValidHcal") << "HcalTestAnalysis:: Initialise "
-                                << "HcalNumberingFromDDD";
+  const HcalDDDSimConstants *hcons = &es.getData(ddconsToken_);
+  edm::LogVerbatim("ValidHcal") << "HcalTestAnalysis:: Initialise HcalNumberingFromDDD";
   numberingFromDDD = new HcalNumberingFromDDD(hcons);
```

Comment:
Just checking...
Previously the code made `new` `numberingFromDDD` and `org` at the begin of the job, and then `numberingFromDDD` was deleted in the destructor.
Now `numberingFromDDD` and `org` are `new`ed at every `beginRun`, but still deleted only once.
Is this the wanted behaviour?

Reply:
I shall take care of this once I get the IB with the merged PR. It is always not straight forward to take care of an old piece of code